frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -84,6 +84,8 @@ global.people.add({
     assert_typeahead_equals("hi emoji :", false);
     assert_typeahead_equals("hi emoji :ta", emoji_list);
     assert_typeahead_equals("hi emoji :da", emoji_list);
+    assert_typeahead_equals("hi emoji :da_", emoji_list);
+    assert_typeahead_equals("hi emoji :da ", emoji_list);
```

Comment:
The `emoji_list` is set globally as `emoji.emojis`. The `compose_content_begins_typeahead` function returns this emoji list (Line 55).
Creating a different list for spaced emojis would mean modifying the `compose_content_begins_typeahead`. Instead, it would be better if the function returns the individual emoji object instead of a list.